src/components/Search/SearchContext.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useCallback, useContext, useMemo, useState} from 'react';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
+import type {SearchContext} from './types';
+
+const defaultSearchContext = {
+    currentSearchHash: -1,
+    selectedTransactionIds: [],
```

Comment:
Same for the other instances below

```suggestion
    selectedTransactionIDs: [],
```